test/e2e/specs/record/helpers.js

```diff
@@ -167,7 +167,12 @@ exports.testPresentation = function (tableParams) {
                     // verify all rows are present
                     chaisePage.recordPage.getRelatedTableRows(displayName).count().then(function(rowCount) {
                         expect(rowCount).toBe(relatedTables[i].data.length);
-                        expect(headings[i]).toBe("- " + displayName + " (" + rowCount + ")");
+                        // The annotation_image table has more rows than the page_size, so its heading will have a + after the row count
+                        if (displayName == tableParams.related_table_name_with_page_size_annotation) {
+                            expect(headings[i]).toBe("- " + displayName + " (" + rowCount + "+)");
+                        } else {
+                            expect(headings[i]).toBe("- " + displayName + " (" + rowCount + ")");
+                        }
```

Comment:
Yeah the if-else is just so this case will pass for the different specs. I have added comments the explain this.